criu/net.c

```diff
@@ -3107,6 +3116,9 @@ static inline int nftables_lock_network_internal(void)
 	snprintf(buf, sizeof(buf), "delete table %s", table);
 	NFT_RUN_CMD(nft, buf);
 err2:
+	fflush(fp);
+	fclose(fp);
```

Comment:
fclose() closes the descriptor that has been returned by log_get_fd(). log_get_fd returns the actual log file descriptor and doesn't dup it, so we should not close it. Do I miss something?

Reply:
Interesting question. I just followed existing code in CRIU. Not sure. Let me check.

Reply:
corrected